### ISSUE TITLE
Block eventually truncates logs

### ISSUE THREAD

**Issue by jblaker** [warpdotdev/Warp#3006]
### Describe the bug

We have some xcodebuild scripts that run for a while and output fairly long logs. It appears that blocks in Warp eventually truncate these logs preventing me from seeing all parts of the log when I need to debug.

### To Reproduce

You'd need a script that runs for a while and generates a lot of logs. For example ours runs xcodebuild (including unit tests) for a large codebase (for iOS and tvOS) and then runs an Xcode project that generates a code coverage report followed by executing some SonarQube commands.  

### Expected behavior

The full build log rather than a truncated log.

### Operating System

MacOS

### Operating System and Version

macOS 13.3 (22E252)

### Warp Version

v0.2023.04.25.08.05.stable_00

### Does this block you from using Warp daily?

No

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Thanks for letting us know @jblaker

We're tracking this issue over at #610 and would appreciate it if you could :+1: and comment there if you're also experiencing the same bug. This helps us gauge the impact of the issue. 

TLDR; it's something we are aware of and working to fix in a pragmatic way that doesn't negatively affect performance.

We'll be closing this as a duplicate.
